src/main/java/org/dsmhack/controller/ProjectController.java

```diff
@@ -59,7 +59,7 @@ public Project getProjectById(@PathVariable String projectId) {
     public ResponseEntity<CheckIn> checkUserIn(@PathVariable String projectId, @RequestBody String userGuid){
         CheckIn checkIn = new CheckIn();
         checkIn.setUserGuid(userGuid);
-        checkIn.setProjGuid(projectGuid);
+        checkIn.setProjGuid(projectId);
```

Comment:
Should this variable be renamed to `projectGuid` ?

Reply:
Yeah, I had fixed all the naming awhile back; but I ended up reverting that commit, lol...